okhttp-clients/src/test/java/com/palantir/conjure/java/okhttp/TestBase.java

```diff
@@ -53,11 +53,6 @@ protected final ClientConfiguration createTestConfig(String... uri) {
     }
 
     protected static Response responseWithCode(Request request, int code) {
-        return new Response.Builder()
-                .request(request)
-                .protocol(Protocol.HTTP_1_1)
-                .code(code)
-                .message("unused")
-                .build();
```

Comment:
@dansanduleac thoughts on some kind of heuristic that detects over 4 dot method invocations and doesn't try to squish them onto one line?